spec/models/occupation_standard_spec.rb

```diff
@@ -199,6 +199,29 @@
     end
   end
 
+  describe ".recently_added" do
+    it "excludes occupations without work processes" do
+      create_list(:occupation_standard, 4)
+      occupation_with_work_proccesses = create(:occupation_standard, :with_work_processes)
+
+      expect(described_class.recently_added).to match [occupation_with_work_proccesses]
+    end
+
+    it "returns up to MAX_RECENTLY_ADDED_OCCUPATIONS_TO_DISPLAY" do
+      create_list(:occupation_standard, 5, :with_work_processes)
+
+      expect(described_class.recently_added.count).to eq described_class::MAX_RECENTLY_ADDED_OCCUPATIONS_TO_DISPLAY
```

Comment:
What do you think about stubbing this to reduce DB count and not making the spec dependent on the constant?

```
stub_const("OccupationStandard::MAX_RECENTLY_ADDED_OCCUPATIONS_TO_DISPLAY", 2)
create_list(:occupation_standard, 3, :with_work_processes)

expect(described_class.recently_added.count).to eq 2
```